service/CommerceParentVariant.js

```diff
@@ -189,18 +193,18 @@ class CommerceParentVariant {
       }
 
       const parentVvs = variantValue[`${this.helper.namespace('Parent_Value_Path__c')}`].split(',')
-      for (let i = properOrderVariants.length - 1; i > 0; i--) {
-        tmpObj[`Variation Attribute Value ${i}`] =
-          this.variantValueMap.get(parentVvs[i - 1])[`${this.helper.namespace('Label__c')}`]
+      for (let i = 0; i < properOrderVariants.length - 1; i++) {
```

Comment:
remove `- 1`?

Reply:
the `-1` needs to be there. I am processing index `0` through the second to last because in the lines right below I process the lowest level because I am iterating through the lowest level variant values and I don't have to go anywhere for the value.